### PR TITLE
add a ReadTheDocs badge

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

**Comment by ctb** [comment]
Shall I merge?

**Comment by mr-c** [comment]
Sure

**Comment by ctb** [comment]
Made mergeable; pls go ahead and merge when tests pass.

**Comment by ctb** [comment]
(Trying to stick with the idea that, barring a release-time exception, the last person to make a chance doesn't get to merge it. :)

**Comment by bocajnotnef** [comment]
@mr-c Should be mergable

File: ChangeLog [hunk 1]
```diff
@@ -377,6 +381,7 @@
    * khmer/_khmermodule.cc: added functions hash_find_all_tags_list and
    hash_get_tags_and_positions to CountingHash objects.
    * tests/test_counting_hash.py: added tests for new functionality.
+>>>>>>> 43c03839ebd8d8f8ff110f5a9450f4eacbce0b9e
```
Comment (mr-c) [inline ChangeLog]: @bocajnotnef ?

Comment (bocajnotnef) [inline ChangeLog]: Sigh. I always miss merge markers.

**Comment by bocajnotnef** [comment]
retest this please

**Comment by bocajnotnef** [comment]
@mr-c I fixed it. :<

**Comment by mr-c** [comment]
Thanks @bocajnotnef !

**Comment by bocajnotnef** [comment]
Of course!

**Pull request merged**